Jamfile

```diff
@@ -472,6 +472,9 @@ feature.compose <invariant-checks>full : <define>TORRENT_EXPENSIVE_INVARIANT_CHE
 feature disk-stats : off on : composite propagated link-incompatible ;
 feature.compose <disk-stats>on : <define>TORRENT_DISK_STATS ;
 
+feature utp-stats : off on : composite propagated link-incompatible ;
+feature.compose <utp-stats>on : <define>TORRENT_UTP_LOG <define>TORRENT_VERBOSE_UTP_LOG ;
```

Comment:
I think utp-log would be a more accurate name. Although, I suppose if verbose logging isn't enabled it's more stats-like